test/unit/specs/vislib/visualizations/tile_map.js

```diff
@@ -11,90 +11,144 @@ define(function (require) {
     require('vislib_fixtures/mock_data/geohash/_rows')
   ];
   var names = ['geojson', 'columns', 'rows'];
-  var mapTypes = ['Scaled Circle Markers', 'Shaded Circle Markers', 'Shaded Geohash Grid', 'Pins'];
+  // TODO: Test the specific behavior of each these
+  var mapTypes = ['Scaled Circle Markers', 'Shaded Circle Markers', 'Shaded Geohash Grid'];
 
   angular.module('TileMapFactory', ['kibana']);
 
-  dataArray.forEach(function (data, i) {
```

Comment:
This was calling every test for every type of map, for 3 scenarios, even those the output didn't change and the purpose was really just to see if it barfed on render. 

I change this to only render each type to check barf status, and moved the other tests to be more appropriately scoped.